### PR TITLE
Adds collector and UI IP as env variables and explorer GraphQL tests

### PR DESCRIPTION
**Pull request by JBAhire**
## Description
This PR, 
- adds collector and UI IP as env variables so can be re-used in various step in CD as well as can be configured easily without code change. 
- Adds GraphQL test for explorer. 
- changing defaults of `HT_ENV` and `HT_KUBE_CONTEXT` to minikube. In case of other deployments, users have to change them manually as mentioned in documentation. 

### Testing
Screenshot 2021-01-20 at 11 14 58 AM

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/hypertrace-cd.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: hypertrace CD
+on:
+  push:
+    branches:
+      - parameterised-e2e
```
Comment (rish691) [inline .github/workflows/hypertrace-cd.yml:5]: What does this mean?

File: .github/workflows/hypertrace-cd.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        uses: actions/checkout@v2
+        with: 
+          ref: parameterised-e2e
+
+      - name: Start minikube
+        run: minikube start
+      - name: Install Hypertrace
+        working-directory: kubernetes
+        run: |
+          kubectl create ns hypertrace
```
Comment (rish691) [inline .github/workflows/hypertrace-cd.yml:24]: The install script would create the namespace

File: .github/workflows/hypertrace-cd.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          minikube logs
+          minikube status
+
+      - name: Waits for some stability
+        run: |
+          sleep 60 # you can decrease it but never increase it
+          kubectl get pods -n hypertrace
+      
+      - name: Set up Minikube tunnel
+        run: minikube tunnel &> /dev/null &
```
Comment (rish691) [inline .github/workflows/hypertrace-cd.yml:40]: Why is this tunnel required?

File: .github/workflows/hypertrace-cd.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      - name: run UI tests
+        working-directory: ./.github/ui-e2e-tests/
+        run: |
+          npm install 
+          npm audit fix
+          CHROMEDRIVER_RELEASE="$(google-chrome --product-version)" && npm run install-web-driver -- --versions.chrome=${CHROMEDRIVER_RELEASE}
+          npx protractor protractor.conf.js --suite smoke --baseUrl "http://${HTUI_IP}:2020"
+
+      - name: Clean up
+        if: ${{ always() }}
```
Comment (rish691) [inline .github/workflows/hypertrace-cd.yml:64]: What does this `if` condition mean?
would it always run this command?

File: .github/workflows/hypertrace-cd.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        run: |
+          minikube delete
+
+      - name: Clean up
```
Comment (rish691) [inline .github/workflows/hypertrace-cd.yml:68]: This can be done in the previous step?

**Comment by JBAhire** [comment]
closing this as this is handled by: https://github.com/hypertrace/hypertrace/pull/189

**Pull request closed without merging**